obs-studio-client/source/utility.hpp

```diff
@@ -48,5 +55,41 @@
 #endif
 #endif
 
+#if defined(_MSC_VER)
+#define FORCE_INLINE __forceinline
+#else
+#define FORCE_INLINE __attribute__((always_inline))
+#endif
+
 #define dstr(s) #s
 #define vstr(s) dstr(s)
+
+static FORCE_INLINE bool ValidateResponse(std::vector<ipc::value> &response)
+{
+        if ((response.size() == 1) && (response[0].type == ipc::type::Null)) {
+                Nan::ThrowError(Nan::New<v8::String>(response[1].value_str).ToLocalChecked());
+                return false;
+        }
+
+        {
+                ErrorCode error = (ErrorCode)response[0].value_union.ui64;
+                if (error != ErrorCode::Ok) {
+                        Nan::ThrowError(Nan::New<v8::String>(response[0].value_str).ToLocalChecked());
+                        return false;
+                }
+        }
+
+        if (!response.size()) {
+                Nan::ThrowError(__FUNCTION_NAME__ ": Failed to make IPC call, verify IPC status.");
```

Comment:
__FUNCTION_NAME__ is a macro and will always show "ValidateResponse" here. For a proper stack identification, use `dbghelp.dll`. Documentation for that is here: https://msdn.microsoft.com/en-us/library/windows/desktop/ms679291(v=vs.85).aspx.